lib/performance_platform/gateway/performance_report.rb

```diff
@@ -0,0 +1,41 @@
+class PerformancePlatform::Gateway::HttpError < StandardError; end
```

Comment:
I wasn't sure about this at first, but I think I like it.

I also wasn't sure about the way it's expressed (here at the top of a file then instantiated in-line), but I can't find a better way to do it - so looks good!